filebeat/tests/integration/testdata/take-over/happy-path-log-input.yml

```diff
@@ -0,0 +1,46 @@
+filebeat.inputs:
+  - type: {{ if .takeOver }}filestream{{ else }}log{{ end}}
+    paths:
+      - {{.testdata}}/take-over/*.log
+{{ if .takeOver }}
+    id: take-over-from-log-input
+    take_over:
+      enabled: true
+    file_identity.fingerprint: ~
+    prospector:
+      scanner:
+        fingerprint:
+          enabled: true
+          length: 64
+        check_interval: 0.1s
+{{ else }}
+    allow_deprecated_use: true
+    scan_frequency: 0.1s
+{{ end }}
+
+  - type: log
+    paths:
+      - {{.testdata}}/take-over/*.txt
+    scan_frequency: 0.1s
+    allow_deprecated_use: true
```

Comment:
If the correct way to take over is not have the log input anymore, why did you keep it in the test?

Reply:
This input is not taken over, only the first input is.

This input is reading the `*.txt` files, the log input we're testing taking over states reads the `*.log` files.

This input is there just to ensure its states are not touched by the take over mode.